cmd/clusterctl/cmd/init.go

```diff
@@ -112,10 +107,7 @@ func init() {
 	initCmd.Flags().BoolVar(&initOpts.validate, "validate", true,
 		"If true, clusterctl will validate that the deployments will succeed on the management cluster.")
 
-	// TODO: Move this to a sub-command or similar, it shouldn't really be a flag.
-	initCmd.Flags().BoolVar(&initOpts.listImages, "list-images", false,
```

Comment:
Let's first deprecate the flag rather than dropping it completely. 
We can deprecate using:
```
initCmd.Flags().MarkDeprecated("list-images", "use 'clusterctl init list-images' instead.")
```


And also add a note in the book about the deprecation and also udpate the clusterctl section of the book.